docs/application/dotnet/getting-started/sec-privileges.md

```diff
@@ -0,0 +1,191 @@
+| `http://tizen.org/privilege/internet` | public |  | The application can access the Internet. This may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/keygrab` | platform |  | The application can read actions involving special keys, such as the volume keys on this or other devices (e.g. TV remote controls), even when it is running in the background. |
+| `http://tizen.org/privilege/keymanager` | public |  | The application can save keys, certificates, and data to, and retrieve and delete them from, password-protected storage. Checking the statuses of certificates while connected to a mobile network may result in additional charges depending on user's payment plan. |
+| `http://tizen.org/privilege/keymanager.admin` | platform |  | The application can lock and unlock password-protected storage, and manage password changes for it. |
+| `http://tizen.org/privilege/led` | public |  | The application can turn LEDs on or off, such as the LED on the front of the device and the camera flash. |
+| `http://tizen.org/privilege/location` | public | Location | The application can use user's location data. |
+| `http://tizen.org/privilege/location.coarse` | public | Location | The application can determine user's approximate location including user's device's Cell ID, LAC (Location Area Code), and TAC (Tracking Area Code). |
+| `http://tizen.org/privilege/location.enable` | platform |  | The application can control user's location service settings. |
+| `http://tizen.org/privilege/mapservice` | public |  | The application can use map services such as Geocoder, Places, and Route (Direction). |
+| `http://tizen.org/privilege/mediacontroller.client` | public |  | The application can receive information about currently playing media from applications that are allowed to send it, and can control those applications remotely. |
+| `http://tizen.org/privilege/mediacontroller.server` | public |  | The application can send information about currently playing media to applications that are allowed to receive it, and can be controlled remotely by those applications. |
+| `http://tizen.org/privilege/mediahistory.read` | public | User history | The application can read the statistics concerning the music and videos played on the device, such as the peak times for playing music or videos. |
+| `http://tizen.org/privilege/mediastorage` | public | Storage | The application can read and write files in media folders. |
+| `http://tizen.org/privilege/message.read` | public | Message | The application can read text and multimedia messages, and any information related to them. |
+| `http://tizen.org/privilege/message.write` | public | Message | The application can write, send, delete, and move text and multimedia messages, download multimedia messages, and change the settings and statuses of messages, such as read or unread. This may result in additional charges depending on user's payment plan. |
```

Comment:
The application can write, send, delete, move text and multimedia messages, download multimedia messages, and change the settings and statuses of messages, such as read or unread.

Reply:
done